api/src/services/user-db.js

```diff
@@ -63,9 +63,11 @@ module.exports = {
     const dbName = module.exports.getDbName(username);
     return db.exists(dbName).then(found => {
       if (!found) {
-        return db.get(dbName)
-          .then(database => database.put(ddoc))
-          .then(() => setSecurity(dbName, username));
+        const database = db.get(dbName);
+        return database.put(ddoc)
+          .then(result => {
```

Comment:
`result` isn't used, right? It's weird that eslint isn't erroring on this...

Reply:
Yeah! I noticed this morning that eslint was way too relaxed. If I get time today I'm going to go through our rules and pair them down as much as possible.